### DISCUSSION TITLE
Logo Ideas

### DISCUSSION THREAD

**Discussion by 3quarterscolon** [TheMowgliMan/Country-Clash#4] [General]
Here’s mine:

![5D1BDBA3-02F0-4420-868E-927563705643](https://user-images.githubusercontent.com/94267030/169667699-bef7af42-43cb-4a2f-a3a0-928cdb670eff.png)

**Comment by TheMowgliMan** [comment]
That's good, I made an Icon we could incorporate into the logo:
![icon](https://user-images.githubusercontent.com/96443213/169668059-ce4d38ee-07e0-4bfa-a059-f11318e594cf.png)

**Reply by TheMowgliMan** [reply]
It's pixel art, download it and zoom in to get a better view.

**Reply by 3quarterscolon** [reply]
Nice. The only problem is that my iPad’s image editor blurs small images when they are scaled up, so I can’t add it right now.

**Reply by TheMowgliMan** [reply]
hmm, alright. That's called filtering, and it makes HD images look nicer, but it's useless tous.

**Reply by 3quarterscolon** [reply]
yeah

**Reply by TheMowgliMan** [reply]
*to us. Anyway, I'm working on the map renderer right now. So far so good.

**Reply by 3quarterscolon** [reply]
Okay. I was going to try to make the starting menus (main menu, map selector, country selector) whenever I can get on my computer.

**Comment by TheMowgliMan** [comment]
Alrighty!

**Comment by ghost** [comment]
I like it, incorporating the globe logo in places that's too long would also be a nice touch.

**Reply by TheMowgliMan** [reply]
An HD Vector logo could also work, considering this game seems to be heading in an HD direction anyway.
